src/main/java/com/suse/salt/netapi/calls/runner/Manage.java

```diff
@@ -27,6 +27,15 @@ private Manage() { }
                 new TypeToken<List<String>>(){});
     }
 
+    public static RunnerCall<List<String>> up(Optional<Integer> timeout,
+            Optional<Integer> gatherJobTimeout) {
```

Comment:
is there a semantical difference between the parameters to `up` and the now generally introduced ones? what happens if you specify both sets of parameters? which one will win?